a/ansible_event-driven-ansible/event-driven-ansible_ubi_8_7.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# ---------------------------------------------------------------------
+#
+# Package       : event-driven-ansible
+# Version       : main
+# Source repo   : https://github.com/ansible/event-driven-ansible/
+# Tested on     : UBI 8.7
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Ashwini Kadam <Ashwini.Kadam@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ---------------------------------------------------------------------
+set -e
+
+PACKAGE_NAME=event-driven-ansible
+PACKAGE_URL=https://github.com/ansible/event-driven-ansible/
+PACKAGE_VERSION=${1:-main}
+PACKAGE_MVN=${PACKAGE_MVN:-"3.8.8"}
+
+yum install java-17-openjdk-devel openssl-devel git wget tar python39-devel.ppc64le gcc rust cargo gcc-c++ cmake.ppc64le systemd-devel -y
+      
+wget https://dlcdn.apache.org/maven/maven-3/$PACKAGE_MVN/binaries/apache-maven-$PACKAGE_MVN-bin.tar.gz
+ls /usr/local
+tar -C /usr/local/ -xzf apache-maven-$PACKAGE_MVN-bin.tar.gz
+mv /usr/local/apache-maven-$PACKAGE_MVN /usr/local/maven
+ls /usr/local
+rm apache-maven-$PACKAGE_MVN-bin.tar.gz
+export M2_HOME=/usr/local/maven
+# update the path env. variable
+export PATH=$PATH:$M2_HOME/bin
+
+export JDK_HOME=/usr/lib/jvm/java-17-openjdk
+export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+    
+#Check if package exists
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME | Removed existing package if any"
+fi
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    exit 0
```

Comment:
This should be may be only exit.. Exit 0 means success state.

Reply:
done